### PR TITLE
[FIX] prevented an NPE when auth callout key was provided empty or non nkey value

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

File: cmd/editauthorization.go [hunk 1]
```diff
@@ -118,8 +118,11 @@ func toPublicKey(s string, kind nkeys.PrefixByte) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if kp == nil {
+		return "", fmt.Errorf("'%s' is not a valid %s key", s, kind)
+	}
 	if err := nkeys.CompatibleKeyPair(kp, kind); err != nil {
-		return "", fmt.Errorf("%s is not a valid %s key", s, kind)
+		return "", fmt.Errorf("'%s' is not a valid %s key", s, kind)
```
Comment (derekcollison) [inline cmd/editauthorization.go:125]: Just %q?

File: cmd/editauthorization_test.go [hunk 2]
```diff
@@ -44,7 +44,7 @@ func Test_EditAuthorizationBadUser(t *testing.T) {
 	_, aPK, _ := CreateAccountKey(t)
 	_, _, err := ExecuteCmd(createEditAuthorizationCallout(), "--auth-user", aPK)
 	require.Error(t, err)
-	require.Equal(t, fmt.Sprintf("%s is not a valid user key", aPK), err.Error())
+	require.Equal(t, fmt.Sprintf("'%s' is not a valid user key", aPK), err.Error())
```
Comment (derekcollison) [inline cmd/editauthorization_test.go:47]: Same

File: cmd/editauthorization_test.go [hunk 3]
```diff
@@ -56,7 +56,17 @@ func Test_EditAuthorizationBadAccount(t *testing.T) {
 	_, aPK, _ := CreateUserKey(t)
 	_, _, err := ExecuteCmd(createEditAuthorizationCallout(), "--allowed-account", aPK)
 	require.Error(t, err)
-	require.Equal(t, fmt.Sprintf("%s is not a valid account key", aPK), err.Error())
+	require.Equal(t, fmt.Sprintf("'%s' is not a valid account key", aPK), err.Error())
```
Comment (derekcollison) [inline cmd/editauthorization_test.go:59]: Same

File: cmd/editauthorization.go [hunk 4]
```diff
@@ -118,8 +118,11 @@ func toPublicKey(s string, kind nkeys.PrefixByte) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if kp == nil {
+		return "", fmt.Errorf("'%s' is not a valid %s key", s, kind)
```
Comment (derekcollison) [inline cmd/editauthorization.go:122]: %q?

File: cmd/editauthorization.go (hunk 1 above)
Comment (aricart) [reply cmd/editauthorization.go:125]: fixed

File: cmd/editauthorization_test.go (hunk 2 above)
Comment (aricart) [reply cmd/editauthorization_test.go:47]: fixed

File: cmd/editauthorization_test.go (hunk 3 above)
Comment (aricart) [reply cmd/editauthorization_test.go:59]: fixed

File: cmd/editauthorization.go (hunk 4 above)
Comment (aricart) [reply cmd/editauthorization.go:122]: fixed

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**